packages/cctp/tests/l-1-usdc-message-transmitter.test.ts

```diff
@@ -1,318 +0,0 @@
-      handleMessageSent(newMessageSentToArb);
-      handleMessageSent(newMessageSentToAvalanche);
-
-      assert.entityCount("MessageSent", 1);
-      assert.fieldEquals(
-        "MessageSent",
-        "0x060000000000000000000000000000000000000000000000000000000000000394c9",
-        "sender",
-        sender.toHexString()
-      );
-      assert.fieldEquals(
-        "MessageSent",
-        "0x060000000000000000000000000000000000000000000000000000000000000394c9",
-        "nonce",
-        "234697", // 394C9
-      );
-      assert.fieldEquals(
-        "MessageSent",
-        "0x060000000000000000000000000000000000000000000000000000000000000394c9",
-        "sourceDomain",
-        "6", // It should be 0, but testing for 0 would always return true if we have wrong decoding
-      );
-      assert.fieldEquals(
-        "MessageSent",
-        "0x060000000000000000000000000000000000000000000000000000000000000394c9",
-        "message",
-        newMessageSentToArb.params.message.toHexString(),
-      );
-      assert.fieldEquals(
-        "MessageSent",
-        "0x060000000000000000000000000000000000000000000000000000000000000394c9",
-        "recipient",
-        "0x7503aad60fd0d205702b0dcd945a1b36c42101b3"
-      );
-      assert.fieldEquals(
-        "MessageSent",
-        "0x060000000000000000000000000000000000000000000000000000000000000394c9",
-        "amount",
-        "1000000" // F4240
-      );
-    });
-
-    test("Older MessageSent event with same (nonce, sourceDomain) is skipped", () => {
-      const newMessageSentToArb = createMessageSentEvent(
-        Bytes.fromHexString(
-          "00000000000000060000000300000000000394C9000000000000000000000000D0C3DA58F55358142B8D3E06C1C30C5C6114EFE800000000000000000000000012DCFD3FE2E9EAC2859FD1ED86D2AB8C5A2F935200000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000007865C6E87B9F70255377E024ACE6630C1EAA37F0000000000000000000000007503AAD60FD0D205702B0DCD945A1B36C42101B30000000000000000000000000000000000000000000000000000000000144B500000000000000000000000009481EF9E2CA814FC94676DEA3E8C3097B06B3A33",
-        ),
-      );
-      
-      const sender = Address.fromString("9481EF9E2CA814FC94676DEA3E8C3097B06B3A33");
-      handleMessageSent(newMessageSentToArb);
-
-      assert.entityCount("MessageSent", 1);
-      assert.fieldEquals(
-        "MessageSent",
-        "0x060000000000000000000000000000000000000000000000000000000000000394c9",
-        "sender",
-        sender.toHexString(),
-      );
-      assert.fieldEquals(
-        "MessageSent",
-        "0x060000000000000000000000000000000000000000000000000000000000000394c9",
-        "nonce",
-        "234697", // 394C9
-      );
-      assert.fieldEquals(
-        "MessageSent",
-        "0x060000000000000000000000000000000000000000000000000000000000000394c9",
-        "sourceDomain",
-        "6", // It should be 0, but testing for 0 would always return true if we have wrong decoding
```

Comment:
we should try to keep this comment somewhere for reading through the tests

can you also check the code again and see if anything needs to be changed to keep using `"6"`?

Reply:
my question is whether we need `sourceDomain` to be `6` in this kind of test values because of what the comment described

if it's irrelevant then i'm ok with it

```typescript
  const newMessageSentToAvalanche = createMessageSentEvent(
    Bytes.fromHexString(
      `000000000000000${sourceDomain}00000001000000000003902D000000000000000000000000D0C3DA58F55358142B8D3E06C1C30C5C6114EFE8000000000000000000000000EB08F243E5D3FCFF26A9E38AE5520A669F4019D000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000007865C6E87B9F70255377E024ACE6630C1EAA37F000000000000000000000000B28CB81B1C50539AE1E941573EBA241E47F6DE5A00000000000000000000000000000000000000000000000000000000000F42400000000000000000000000003A554156AEA1921ABB277F63D6109CA81B530A3E`
    )
  );
```

Reply:
it's not relevant anymore, no